config/environments/development.rb

```diff
@@ -14,22 +16,21 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  # if Rails.root.join('tmp', 'caching-dev.txt').exist?
-  #   config.action_controller.perform_caching = true
-  #
-  #   config.cache_store = :memory_store
-  #   config.public_file_server.headers = {
-  #     'Cache-Control' => "public, max-age=#{2.days.to_i}"
-  #   }
-  # else
-  #   config.action_controller.perform_caching = false
-  #
-  #   config.cache_store = :null_store
-  # end
-
-  config.action_controller.perform_caching = true
```

Comment:
Now in Rails 6 we have the `rails dev:cache` which changes the way we enable caching in development on demand [more here](https://www.akshaykhot.com/rails-enable-caching-in-development/).

I tested that after enabling caching with that command the partial caching examples still work fine.

@devpost-mzheng  Do we need to enable it by default as the server boots up?

Reply:
how was it enabled?